### PR TITLE
Notify admins when users join [WIP]

### PR DESCRIPTION
**Pull request by kdoh**
implements #764
- [x] email template
- [x] email sending logic

### CODE REVIEW

**Comment by salsakran** [comment]
@camsaul mind wiring up this email when you get a free moment?

**Comment by camsaul** [comment]
ya I can do that.

File: src/metabase/email/user_joined_notification.mustache [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{{> metabase/email/_header }}
+  <div style="padding: 2em 4em; border: 1px solid #dddddd; border-radius: 2px; background-color: white; box-shadow: 0 1px 2px rgba(0, 0, 0, .08); text-align: center;">
+    <div style="padding-bottom: 1em;">
+      <h2 style="font-weight: normal; color: #4C545B;line-height: 1.65rem;">
+        {{joinedUserName}}
+        {{#jonedViaSSO}}created a Metabase account.{{/joinedViaSSO}}
```
Comment (camsaul) [inline src/metabase/email/user_joined_notification.mustache]: `jonedViaSSO`? 😉 

I'll fix that

**Comment by camsaul** [comment]
@kdoh what's the difference between `joinedViaSSO` and `sso`?

**Comment by kdoh** [comment]
@camsaul Only my derpy oversight. Forgot to tweak the var name after I changed it to be more descriptive.

**Comment by camsaul** [comment]
i'll change it.

**Comment by camsaul** [comment]
`joinedUserName` and `joinUserName` are both the same thing too right?

**Comment by camsaul** [comment]
slash what subject do you want for the email @kdoh

**Comment by kdoh** [comment]
Yeesh. I really was thorough with those var names wasn't I. I'm thinking subject wise for when the user creates an account on their own via sso "<name> created a Metabase account" and for an invite being accepted we'd go with "<name> accepted your Metabase invite." @mazameli do those sound alright to you?

**Comment by camsaul** [comment]
uh I'm not doing anything with accepted invites here ...?

**Comment by camsaul** [comment]
just admin notifications

**Comment by kdoh** [comment]
@camsaul It's the same thing. When a new user joins they join either via SSO account creation or as the result of an admin invite.

**Comment by camsaul** [comment]
got it

**Comment by camsaul** [comment]
so when a new user accepts an invite we notify the admin who invited them, when someone joins via SSO we notify the site admin?

**Comment by mazameli** [comment]
@kdoh, yeah, those subjects were what I was thinking, too.

**Comment by kdoh** [comment]
@camsaul Yup. In the case of SSO users joining since there may be multiple admin(s) we should notify all admins for now. This won't scale forever and eventually we may need to allow for an opt out as discussed in #764 but should probably be ok in the short term.

**Comment by camsaul** [comment]
I'm guessing `joinedUserEditUrl`== `joinUrl` as well

**Comment by camsaul** [comment]
k this should be g2g if you want to verify @kdoh.

**Comment by kdoh** [comment]
@camsaul The templates look good so once you're happy on your end I think this should be good to go.

**Comment by camsaul** [comment]
K i'll merge once the tests wrap up

**Pull request merged**